tools/update_required_branch_checks.py

```diff
@@ -43,6 +43,9 @@
     # Excludes blackbox, web platform, and unit tests run on-device.
     '_on_device_',
 
+    # Excludes evergreen as blackbox test.
+    'evergreen-as-blackbox_test',
```

Comment:
I'm assuming this applies as a regex or wildcard, right ? in any case, lgtm for now

Reply:
Yes, it does a substring match